### PR TITLE
Adding SIG related information to Governance documentation

### PR DESCRIPTION
**Pull request by melba-lopez**
original Issue: https://github.com/slsa-framework/slsa/issues/453

Added 1.6. Special Interest Group Members to Governance.md
Added Reporting Violations section with associated contacts to Code_of_Conduct.md
Added 3 files for SIG charters as they stand today.
Added Special Interest Groups section to Readme

Updated Readme.md

Related issues, PRs, docs:
https://github.com/slsa-framework/slsa/issues/384
https://github.com/slsa-framework/slsa-proposals/tree/main/0002
https://docs.google.com/document/d/1L1gEJMBIvE0IbpFi23FOUByDYlItSYPPJmKdhvJQYsg/edit#
@melba-lopez

### CODE REVIEW

File: 10._Positioning_Charter.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+- Identifying new use cases 
+- Identifying new personas
+- Identifying opportunities for SLSA expansion
+- Identifying opportunities for shifting SLSA strategy/scope
+- Enabling additional OpenSSF participation
+- Improving security of our open source communities/industry
+
+## Operating Goals
+In addition to the Collaboration/Workstream process documented in [SLSA SIGs document](https://docs.google.com/document/d/1L1gEJMBIvE0IbpFi23FOUByDYlItSYPPJmKdhvJQYsg/edit#heading=h.1hce59kd4nn0), the following are our short term/long term goals:
```
Comment (MarkLodato) [inline 10._Positioning_Charter.md:60]: nit: can we make this charter stand-alone without linking to original proposal? It's not clear to me what, if anything, one is supposed to read in that doc related to this section.

Same below.

**Review by MarkLodato — CHANGES_REQUESTED** [review]
This is great. Thank you, Melba! Overall LGTM. My comments are mostly nitpicks and suggestions, plus one important question on scope.

File: Readme.md [hunk 2]
```diff
@@ -22,3 +22,24 @@ To contact the steering committee:
...
+
+Specification
+-   [Mark Lodato](https://github.com/MarkLodato) - Google
+-   [Joshua Lock](https://github.com/joshuagl) - VMware
+-   Via slack: [#slsa-specification](https://openssf.slack.com/archives/C03NUSAPKC6)
+-   Via email: slsa-steering-committee@googlegroups.com  
+
+Tooling
+-   [Mike Lieberman](https://github.com/mlieberman85) - Kusari/CNCF
+-   [Eric Tice]
```
[the remark this answers is not in the document]
Comment (melba-lopez) [reply Readme.md:37]: fixed; will commit shortly

File: Readme.md [hunk 3]
```diff
@@ -22,3 +22,24 @@ To contact the steering committee:
 
 -   On GitHub: `@slsa-framework/slsa-steering-committee`
 -   Via email: slsa-steering-committee@googlegroups.com
+
+## Special Interest Groups
+To contact the various SIG Leads:
+
+Specification
```
[the remark this answers is not in the document]
Comment (melba-lopez) [reply Readme.md:29]: done; will commit shortly.

File: 11._Tooling_Charter.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Please see [SLSA SIGs document](https://docs.google.com/document/d/1L1gEJMBIvE0IbpFi23FOUByDYlItSYPPJmKdhvJQYsg/edit#heading=h.1hce59kd4nn0) for more information.
```
[the remark this answers is not in the document]
Comment (melba-lopez) [reply 11._Tooling_Charter.md:1]: done will commit shortly

File: 9._Specification_Charter.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+Please see [SLSA SIGs document](https://docs.google.com/document/d/1L1gEJMBIvE0IbpFi23FOUByDYlItSYPPJmKdhvJQYsg/edit#heading=h.1hce59kd4nn0) for more information.
```
[the remark this answers is not in the document]
Comment (melba-lopez) [reply 9._Specification_Charter.md:1]: done; will commit shortly

File: 8._Code_of_Conduct.md [hunk 6]
```diff
@@ -79,3 +79,9 @@ Community Impact Guidelines were inspired by [Mozilla's code of conduct enforcem
 
 For answers to common questions about this code of conduct, see the FAQ at
 https://www.contributor-covenant.org/faq. Translations are available at https://www.contributor-covenant.org/translations.
+
+## Reporting Violations
+To report instance(s) of unacceptable behavior, please contact:
+-   [Joshua Lock](https://github.com/joshuagl) - VMware
+-   [Melba Lopez](https://github.com/melba-lopez) - IBM
+-   Or any [Steering Committee Member](https://github.com/slsa-framework/governance#steering-committee)
```
[the remark this answers is not in the document]
Comment (melba-lopez) [reply 8._Code_of_Conduct.md:87]: done; will commit shortly

File: 10._Positioning_Charter.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
+# Positioning SIG Charter
+
+## Mission
```
[the remark this answers is not in the document]
Comment (melba-lopez) [reply 10._Positioning_Charter.md:3]: Done; you will have to double-check I didn't miss anything once i've committed.

File: 10._Positioning_Charter.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
+# Positioning SIG Charter
```
[the remark this answers is not in the document]
Comment (melba-lopez) [reply 10._Positioning_Charter.md:1]: Done; will commit shortly

**Pull request closed without merging**